src/test/java/tests/contentTesting/LeapFrogContentValidationTest.java

```diff
@@ -0,0 +1,43 @@
+package tests.contentTesting;
+
+import data.GameData;
+import dataFactory.GameDataMapper;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import pages.LeapFrog.LeapFrogStorePage;
+import utils.DataComparator;
+import utils.ExcelUtils;
+
+import java.io.File;
+import java.net.URL;
+import java.util.List;
+
+import static org.testng.Assert.assertNotNull;
+
+public class LeapFrogContentValidationTest extends LeapFrogBaseTest {
+    int totalPages;
+    URL resource;
+    private LeapFrogStorePage storePage;
+    private List<GameData> expectedData;
+    private List<GameData> actualData;
+
+    @BeforeMethod
+    public void setup() throws Exception {
+
+        storePage = new LeapFrogStorePage();
+
+        resource = getClass().getClassLoader().getResource("data/leapfrog-games.xlsx");
```

Comment:
save your file path in a constant class/variable, then pass it to ExcelUtils.loadExcelData
--> Help us reduce number line of code in setUp section here